### DISCUSSION TITLE
Need help in releasing BETA version of Dynamic Autocomplete

### DISCUSSION THREAD

**Discussion by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#181] [General]
Hey Folks,

Please let me know who is available this week to release BETA of autocomplete #126. We can get on a meet call
cc @PradyumnaKrishna @pranavbaburaj @peb-peb

**Comment by lainq** [comment]
Ill try to be there :D

**Reply by GouravSardana** [reply]
🙃

**Comment by peb-peb** [comment]
I'll be there! :)

**Comment by PradyumnaKrishna** [comment]
I will be there too.
